test/unit/mdc-top-app-bar/mdc-top-app-bar.test.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {assert} from 'chai';
+import bel from 'bel';
+import domEvents from 'dom-events';
+import td from 'testdouble';
+
+import {MDCTopAppBar} from '../../../packages/mdc-top-app-bar';
+
+function getFixture() {
+  return bel`
+    <div>
+        <header class="mdc-top-app-bar">
+      <div class="mdc-top-app-bar__row">
+        <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start">
+          <a href="#" class="material-icons mdc-top-app-bar__menu-icon">menu</a>
+          <span class="mdc-top-app-bar__title">Title</span>
+        </section>
+        <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end" 
+        role="top-app-bar">
+          <a href="#" class="material-icons mdc-top-app-bar__icon" aria-label="Download" alt="Download">
+          file_download</a>
+          <a href="#" class="material-icons mdc-top-app-bar__icon" aria-label="Print this page" alt="Print this page">
+          print</a>
+          <a href="#" class="material-icons mdc-top-app-bar__icon" aria-label="Bookmark this page" 
+          alt="Bookmark this page">bookmark</a>
+          <div class="mdc-menu-anchor">
+            <div class="mdc-menu" tabindex="-1" id="demo-menu">
+              <ul class="mdc-menu__items mdc-list" role="menu" aria-hidden="true" style="transform: scale(1, 1);">
+              </ul>
+            </div>
+          </div>
+        </section>
+      </div>
+    </header>
+      <main class="mdc-top-app-bar-fixed-adjust">
+      </main>
+    </div>
+  `;
+}
+
+function setupTest() {
+  const fixture = getFixture();
+  const root = fixture.querySelector('.mdc-top-app-bar');
+  const adjust = fixture.querySelector('.mdc-top-app-bar-fixed-adjust');
+  const component = new MDCTopAppBar(root);
+  return {root, adjust, component};
+}
+
+suite('MDCTopAppBar');
+
+test('attachTo initializes and returns an MDCTopAppBar instance', () => {
+  assert.isOk(MDCTopAppBar.attachTo(getFixture()) instanceof MDCTopAppBar);
+});
+
+test('adapter#hasClass returns true if the root element has specified class', () => {
+  const {root, component} = setupTest();
+  root.classList.add('foo');
+  assert.isOk(component.getDefaultFoundation().adapter_.hasClass('foo'));
```

Comment:
Nit: I know we do this a bunch of places, but I'd advocate using `isTrue` and `isFalse` specifically given that the API is supposed to return a boolean.